src/middleware/error.logging.ts

```diff
@@ -41,8 +41,7 @@ export const errorLoggingExpress: ErrorRequestHandler = function errorLoggingExp
     query: _loggedNonEmpty(req.query),
     body: _loggedNonEmpty(req.body),
   }, identity); // eg. `compact`
-
-  const { statusCode } = (err as any); // "Property 'FOO' does not exist on type 'Error'."
+  const { statusCode } = ((err as any).extensions?.props)? (err as any).extensions.props : (err as any); // "Property 'FOO' does not exist on type 'Error'."
```

Comment:
the line has gotten too long

```suggestion
  // "Property 'FOO' does not exist on type 'Error'."
  const { statusCode } = ((err as any).extensions?.props)? (err as any).extensions.props : (err as any); 
```